corehq/apps/app_manager/templates/app_manager/partials/releases/releases_table.html

```diff
@@ -174,7 +174,7 @@ <h4 class="panel-release-title">
           <span data-bind="text: menu_item_label()"></span>
           <!--/ko-->
           <span data-bind="if: !built_with.signed()">{% trans "Unsigned Jar" %}</span>
-          {% if request|toggle_enabled:"APPLICATION_ERROR_REPORT" %}
+          {% if request|request_has_privilege:"APPLICATION_ERROR_REPORT" or user.is_superuser or user.is_dimagi %}
```

Comment:
Good call 👍

Reply:
Should we use a single DRY context variable here from [below](https://github.com/dimagi/commcare-hq/pull/33078/files#diff-4cc8f9fdea79e14a71e47e35375788b593bec4e61801e130270f5d9ad8bfed7dR177) so that the this logic is in a single place?